schedule/containers/sle_image_on_sle_host.yaml

```diff
@@ -4,12 +4,14 @@ description:    >
   Extra tests about software in containers module
 schedule:
   - installation/bootloader_start
+  - boot/boot_to_desktop
```

Comment:
how this related to rootless containers ?

Reply:
it is not actually. and it shouldnt be there.